draft-tuexen-opsawg-pcapng.xml

```diff
@@ -2088,6 +2088,19 @@ Section Header
             are long-term secrets, users SHOULD only store non-production keys,
             or ensure proper protection of the pcapng file.</t>
 
+            <t hangText="0x5a4e574b:"><vspace blankLines="0"/>ZigBee NWK Key.
```

Comment:
So these are binary keys, right?

Reply:
Yes these are binary keys. These are same as the "Key" in the Pre-configured Keys dialog in the ZigBee Preferences in the Wireshark UI. This is one of two sources for these keys. The other is a ZigBee APS Transport Key frame that contains the NWK key.

Reply:
I might add "binary" after "16 octet" in both places, just so nobody thinks it's a textual representation in hex or something such as that.

Does "R21" mean ZigBee Document 05-3474-21?  If so, I'm not seeing anything in 2.2.5.1 about the APS key.

And it says "Each 16 octet AES-128 key secures frames exchanged between a pair of network nodes."  If that means there's more than one APS key, one for each node pair, then is the ZigBee APS Key item a single key, so that you have multiple such items, or a list of keys, and, in either case, how do you associate an APS Key with a pair of endpoints?

Reply:
Thanks for the feedback! I've made some changed to address your comments. Hopefully they are addressed.